pyproj/_geod.pyx

```diff
@@ -146,6 +146,63 @@ cdef class Geod:
                     latbuff.data[iii] = _DG2RAD * plat2
                     azbuff.data[iii] = _DG2RAD * pazi2
 
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def _fwd_point(
+        self,
+        object lon1in,
+        object lat1in,
+        object az1in,
+        object s12in,
+        bint radians=False,
+    ):
+        """
+        Scalar optimized function
+        forward transformation - determine longitude, latitude and back azimuth
+        of a terminus point given an initial point longitude and latitude, plus
+        forward azimuth and distance.
+        if radians=True, lons/lats are radians instead of degrees.
+        """
+        cdef double plon2, plat2, pazi2, lon1, lat1, az1, s12
+
+        # We do the type-checking internally here rather than declaring in the function
+        # due to automatically casting length-1 arrays to float that we don't want
+        for x in (lon1in, lat1in, az1in, s12in):
+            if not isinstance(x, (float, int)):
+                raise TypeError(f"Function only accepts scalar input")
+        lon1 = <double>lon1in
+        lat1 = <double>lat1in
+        az1 = <double>az1in
+        s12 = <double>s12in
+
+        with nogil:
+            if radians:
+                lon1 = _RAD2DG * lon1
+                lat1 = _RAD2DG * lat1
+                az1 = _RAD2DG * az1
+
+            geod_direct(
+                &self._geod_geodesic,
+                lat1,
+                lon1,
+                az1,
+                s12,
+                &plat2,
+                &plon2,
+                &pazi2,
+            )
+            # back azimuth needs to be flipped 180 degrees
+            # to match what PROJ geod utility produces.
+            if pazi2 > 0:
```

Comment:
Mind rebasing from master and replacing this if/elif with:
```python
           pazi2 = _reverse_azimuth(pazi2, factor=180)
```

Reply:
Yep, should be rebased and squashed down to a single commit now.